KubeArmor/common/common.go

```diff
@@ -221,8 +221,10 @@ func GetCommandOutputWithErr(cmd string, args []string) (string, error) {
 	}
 
 	go func() {
-		defer stdin.Close()
-		_, _ = io.WriteString(stdin, "values written to stdin are passed to cmd's standard input")
+		defer func() {
+			_ = stdin.Close()
+		}()
```

Comment:
i don't think there's any point in doing this since it doesn't handle the warning just bypasses it.

We should capture and log the error imo

Reply:
https://github.com/securego/gosec/issues/512#issuecomment-675286833 solution for reference